functions/concepts/requests/index.js

```diff
@@ -16,16 +16,17 @@
 
 // [START functions_concepts_requests]
 const fetch = require('node-fetch');
+const functions = require('@google-cloud/functions-framework');
 
 /**
  * HTTP Cloud Function that makes an HTTP request
  *
  * @param {Object} req Cloud Function request context.
  * @param {Object} res Cloud Function response context.
  */
-exports.makeRequest = async (req, res) => {
+functions.http('makeRequest', async (req, res)  => {
```

Comment:
This should fix the lint error
```suggestion
functions.http('makeRequest', async (req, res) => {
```